Dockerfile

```diff
@@ -108,11 +109,13 @@ RUN docker-php-ext-install -j$(nproc) zip
 FROM php:7.3.10-fpm-alpine3.10 AS fpm-alpine-base
 RUN apk add --no-cache \
         bash \
+        freetype \
         haveged \
         icu \
+        libldap \
         libpng \
         libzip \
-        freetype && \
+        openldap-dev && \
```

Comment:
is `libldap` AND `openldap` needed? (duplicate?)
why the `dev` lib in the alpine base build? is there a `openldap`-only available? (should be much smaller in footprint)

Reply:
I was able to remove the openldap package completely.